annif/backend/maui.py

```diff
@@ -18,6 +18,26 @@ class MauiBackend(backend.AnnifBackend):
 
     TRAIN_FILE = 'maui-train.jsonl'
 
+    @property
+    def is_trained(self):
+        params = self._get_backend_params(None)
+        try:
+            resp = requests.get(self.tagger_url(params))
+            resp.raise_for_status()
+        except requests.exceptions.RequestException as err:
+            self.warning("HTTP request failed: {}".format(err))
+            return None
+        try:
+            return resp.json()['is_trained']
+        except ValueError as err:
+            self.warning("JSON decode failed: {}".format(err))
+            return None
+
+    @property
+    def modification_time(self):
+        # TODO Should mtime of Maui project always be None or how to get it?
```

Comment:
You can get the training time (as well as `is_trained`) from the `train` endpoint, e.g. `/mauiserver/maui-en/train` which returns JSON like this:

```json
{"service_status":"ready",
 "completed":true,
 "documents":693,
 "skipped":0,
 "start_time":"2020-05-26T15:35:03.680+03:00",
 "runtime_millis":137488,
 "end_time":"2020-05-26T15:37:21.168+03:00",
 "is_trained":true,
 "training_status":"ready"}
```

I think the `end_time` is the most relevant.